.github/workflows/release-native-bundle.yml

```
@@ -0,0 +1,119 @@
+name: release-native-bundle
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)